model/BookletConfigService.php

```diff
@@ -274,6 +277,16 @@ protected function formatValue($format, $value) {
 
 
     /**
+     * Generates a string that can be used with sprintf()
+     * @return string
+     */
+    protected function getPageFormat() {
+        $format = $this->getOption(self::OPTION_PAGE_FORMAT);
+        return $format ? $format : '%s/%s';
```

Comment:
Maybe should it be better to use a proper constant instead of magic value for the default?